salt/docs/config/nginx.docs-backend.conf

```diff
@@ -56,15 +56,15 @@ server {
         return 301 https://$host/devguide/documenting.html#building-the-documentation;
     }
 
-    # Map toplevel URIs to Python 2 docs.
+    # Map toplevel URIs to Python 3 docs.
     location ~ ^/((archives|c-api|distutils|extending|faq|howto|install|library|reference|tutorial|using|whatsnew|_images|_sources|_static)(/.*)?)$ {
-        return 301 https://$host/2/$1;
+        return 301 https://$host/3/$1;
     }
     location ~ ^/(about|bugs|contents|copyright|download|genindex.*|glossary|index|license|py-modindex|search).html$ {
-        return 301 https://$host/2/$1.html;
+        return 301 https://$host/3/$1.html;
     }
     location ~ ^/(searchindex.js|objects.inv)$ {
-        return 301 https://$host/2/$1;
+        return 301 https://$host/3/$1;
```

Comment:
These are probably mostly fetched by automated tools and I think should remain pointing to 2.